### PR TITLE
HEC-1947: agents page

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: test/uk/gov/hmrc/hecapplicantfrontend/controllers/StartControllerSpec.scala [hunk 1]
```diff
@@ -761,6 +744,27 @@ class StartControllerSpec extends ControllerSpec with AuthSupport with SessionSu
         }
       }
 
+      "redirect to the 'Agents not supposed' page" when {
```
Comment (nubz) [inline test/uk/gov/hmrc/hecapplicantfrontend/controllers/StartControllerSpec.scala:747]: typo in "supposed"

File: conf/messages [hunk 2]
```diff
@@ -420,3 +412,9 @@ cannotSendVerificationPasscode.p2 = You can <a href="{0}" class="govuk-link">go
 
 #Email verification passcode email
 emailVerification.passcodeEmail.serviceName = HMRC tax check service
+
+#Agents not supposed
```
Comment (nubz) [inline conf/messages:416]: typo in "supposed"

**Review by nubz — COMMENTED** [review]
couple of typos

File: conf/messages (hunk 2 above)
Comment (AndyHWChung) [reply conf/messages]: whoops, fixed it just now

**Pull request merged**